lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -477,8 +478,52 @@ def __get_template_with_version(self):
 
         return template
 
+    def __get_storage_domain(self, disk_id):
+        storage_domain = None
+
+        disks = self._module.params.get('disks', list())
+
+        for disk in disks:
+            if disk_id == disk.get('id'):
```

Comment:
The idea here is to specify which disk go where see example:
  ovirt_vms:
    name: vm_test
    cluster: clu1
    template: template01
    operating_system: other_linux
    type: server
    cpu_cores: 1
    cpu_sockets: 1
    state: stopped
    clone: True
    timeout: 600
    disks:
      - id: disk_id1
        storage_domain: my_nfs_1
     -  id: disk_id1
        storage_domains: my_nfs_3

i would prefer not to decalre any disk and only storage domain so all disks related to template will go to spesfic storage domain ... 
but you said there is a use case to split disks of templates on multiple storage domains .... 

please tell me what you suggest... and i will implement it ...